common/constants.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+# HOTSOS GLOBALS
+VERBOSITY_LEVEL = int(os.environ.get('VERBOSITY_LEVEL', 0))
+DATA_ROOT = os.environ.get('DATA_ROOT', '/')
+
+JUJU_LOG_PATH = os.path.join(DATA_ROOT, "var/log/juju")
```

Comment:
I like this but think it needs to be done for all plugins. I will fix this separately so that you can pull it into your patch.

Reply:
See https://github.com/dosaboy/hotsos/commit/da484b58d975331518993935f6e9448569178650

Reply:
done.